server-api/data/length.ts

```diff
@@ -2,8 +2,56 @@ import { Units } from "../models/converter.model";
 
 const units: Units = {};
 
-units['m'] = {unit:'m', base: 'm', name: 'meter', type: 'length', factor: 1};
-units['cm'] = {unit:'cm', base: 'm', name: 'centimeter', type: 'length', factor: 0.01};
-units['mm'] = {unit:'mm', base: 'm', name: 'milimeter', type: 'length', factor: 0.001};
+units["yard"] = {
```

Comment:
@andmiguedev  make the name of the property (units['yd']) the unit not the name

Reply:
Check last changes under Done, I have updated key to be the abbreviation and accurate factor for current lengths